lib/errata/errata_requests.py

```diff
@@ -189,10 +220,33 @@ def format_advisory_data(advisory_data):
 
     final_response["bugs"] = bug_details
     bug_summary_array = []
+    for key in jira_issue_summary:
+        if total_jira_issues == 0:
+            bug_summary_array.append({
+                "bug_status": key,
+                "count": jira_issue_summary[key],
+                "percent": 0
+            })
+        else:
+            bug_summary_array.append({
+                "bug_status": key,
+                "count": jira_issue_summary[key],
+                "percent": round((jira_issue_summary[key] / total_jira_issues) * 100, 2)
+            })
+
     for key in bug_summary:
-        bug_summary_array.append({"bug_status": key,
-                                  "count": bug_summary[key],
-                                  "percent": round((bug_summary[key] / total_bugs) * 100, 2)})
+        if total_bugs == 0:
+            bug_summary_array.append({
+                "bug_status": key,
+                "count": bug_summary[key],
+                "percent": 0
+            })
+        else:
+            bug_summary_array.append({
+                "bug_status": key,
+                "count": bug_summary[key],
+                "percent": round((bug_summary[key] / total_bugs) * 100, 2)
+            })
```

Comment:
Also now that both JIRA and bugzilla stores bugs, it might be a good idea to rename the variables and say `jira_bugs` and `bugzilla_bugs` respectively. This can also be a refinement PR if you prefer

Reply:
good idea! Will do it and amend it in this PR